systemjs.conf.js

```diff
@@ -0,0 +1,45 @@
+var SystemConfig = (function() {
+    // List your node_modules packages here
+    var packages = [
+        'angular-material',
+        'angular-mocks',
+        'ng-metadata',
+        'reflect-metadata'
+    ];
+
+    var config = {
+        baseUrl: '.',
+        paths: {
+            'npm:*': 'node_modules/*'
+        },
+        map: {
+            "css": "npm:systemjs-plugin-css/css.js",
+            "html": "npm:systemjs-plugin-text/text.js",
```

Comment:
Please, try change this line with:
```typescript
"html": "npm:systemjs-text/text.js", // USE OFFICAL system-text  plugin published to npm *this may avoid traceour dep!
```
and remove the traceour dependency, this may works to correctly load the template html, but without using traceour.
Can you give it a try?